mkt/webapps/tests/test_fakedata.py

```diff
@@ -25,7 +25,7 @@ def test_smallset(self):
         ok_(not any(appname[-1].isdigit() for appname, cat in data))
 
     def test_bigset(self):
-        size = 300
+        size = 1500
```

Comment:
Good idea, this should work just fine now that I've simplified things a bit; the original version of this code was mainly intended for small sets so I didn't look too close at the bigger cases.